examples/text-generation/requirements_lm_eval.txt

```diff
@@ -1 +1 @@
-https://github.com/polisettyvarma/lm-evaluation-harness/archive/3cdc8daadad9f4559ae6cdfae96f1d83d6b3c1f4.zip
\ No newline at end of file
+git+https://github.com/EleutherAI/lm-evaluation-harness.git@master
```

Comment:
@polisettyvarma please specify a compatible version (using semantic versioning) or use a branch-name or a working commit

Reply:
https://github.com/EleutherAI/lm-evaluation-harness/commits/master/ seems already frozen and no updates from past 7 to 8 months.

Reply:
No matter that please follow the best practices!